### PR TITLE
Allow to create story without an epic

### PR DESCRIPTION
**Pull request by j-martin**
While it's probably a good idea to assign epics to new stories,
depending on the methodology used by the team not everything may
warrant being attached to an epic. E.g. one-off task and what
not.

### CODE REVIEW

File: org-clubhouse.el [hunk 1]
```diff
@@ -520,7 +520,7 @@ If set to nil, will never create stories with labels")
 (defun org-clubhouse-prompt-for-epic (cb)
   (ivy-read
    "Select an epic: "
-   (-map #'cdr (org-clubhouse-epics))
+   (-map #'cdr (append '((nil . "*** NONE ***")) (org-clubhouse-epics)))
```
Comment (j-martin) [inline org-clubhouse.el]: The exact term is TBD.

File: org-clubhouse.el [hunk 2]
```diff
@@ -665,9 +665,10 @@ If the epics already have a CLUBHOUSE-EPIC-ID, they are filtered and ignored."
                          (number-to-string story-id)))
 
       (org-set-property "clubhouse-epic"
-                        (org-make-link-string
-                         (org-clubhouse-link-to-epic epic-id)
-                         (alist-get epic-id (org-clubhouse-epics))))
+        (if epic-id (org-make-link-string)
+          (org-clubhouse-link-to-epic epic-id)
+          (alist-get epic-id (org-clubhouse-epics))
+          "None"))
```
Comment (j-martin) [inline org-clubhouse.el]: @glittershark the formatting is obviously wrong. I was wondering which tool you are using to format+config your code. Parinfer on my side wants to reformat everything.

File: org-clubhouse.el (hunk 1 above)
Comment (glittershark) [reply org-clubhouse.el]: Let's just say "No Epic" here

File: org-clubhouse.el [hunk 3]
```diff
@@ -432,7 +432,7 @@ If set to nil, will never create stories with labels")
   (org-clubhouse-fetch-as-id-name-pairs "projects"))
 
 (defcache org-clubhouse-epics
-  "Returns projects as (project-id . name)"
+  "Returns epics as (epic-id . name)"
```
Comment (glittershark) [inline org-clubhouse.el]: oops, thanks!

File: org-clubhouse.el [hunk 4]
```diff
@@ -663,11 +663,11 @@ If the epics already have a CLUBHOUSE-EPIC-ID, they are filtered and ignored."
                         (org-make-link-string
                          (org-clubhouse-link-to-story story-id)
                          (number-to-string story-id)))
-
-      (org-set-property "clubhouse-epic"
-                        (org-make-link-string
-                         (org-clubhouse-link-to-epic epic-id)
-                         (alist-get epic-id (org-clubhouse-epics))))
+ (when epic-id
```
Comment (glittershark) [inline org-clubhouse.el]: indentation looks off here

Comment (j-martin) [reply org-clubhouse.el]: Yep, as stated by https://github.com/glittershark/org-clubhouse/pull/24#discussion_r368311922

My indentation config is whack. Which mode do you use/configured of elisp?

Comment (glittershark) [reply org-clubhouse.el]: I just go with whatever `lisp-indent-line` in emacs does automatically.

Comment (j-martin) [reply org-clubhouse.el]: Mistery solved! Thanks! :)

**Pull request merged**